ext/native/math/lin/matrix4x4.h

```diff
@@ -111,6 +111,34 @@ class Matrix4x4 {
 		zz = 1.0f; 
 		ww = 1.0f;
 	}
+	// Exact angles to avoid any artifacts.
+	void setRotationZ90() {
+		empty();
+		float c = 0.0f;
+		float s = 1.0f;
+		xx = c;		xy = s;
```

Comment:
nit: Prefer against tabs after first non-tab?

-[Unknown]

Reply:
Agreed, I'll fix that and delete some unused code.